.github/workflows/ci-e2e-checks.yaml

```diff
@@ -102,14 +102,18 @@ jobs:
         make build-single-manifest
         echo "Install ImageSwap"
         make install
-        kubectl wait --for=condition=Ready pods -l app=imageswap -n imageswap-system
+        sleep 60
+        kubectl describe pods -l app=imageswap -n imageswap-system
+        kubectl logs -l app=imageswap -c imageswap-init -n imageswap-system
+        kubectl -n imageswap-system get csr -o yaml
+        #kubectl wait --for=condition=Ready --timeout=200s  pods -l app=imageswap -n imageswap-system
```

Comment:
I'd guess this is from testing in your fork. I think it just needs to be cleaned up

Reply:
Cleaned!